src/components/icon/icon.test.tsx

```diff
@@ -172,8 +160,47 @@ describe('OuiIcon', () => {
       });
       const component = mount(<OuiIcon type="accessibility" />);
       expect(
-        component.find('OuiIcon[type="accessibility"] > OuiIconEmpty').length
+        component.find('OuiIcon[type="accessibility"] > OuiIconBeaker').length
       ).toBe(1);
     });
   });
+
+  describe('render different types of icons', () => {
+    it('renders icon when type is in cache', () => {
+      appendIconComponentCache({
+        videoPlayer: OuiIconVideoPlayer,
+      });
+      const component = mount(<OuiIcon type="videoPlayer" />);
+      expect(prettyHtml(component.html())).toMatchSnapshot();
+    });
+
+    it('renders custom svg from absolute url', () => {
+      const component = mount(
+        <OuiIcon type="https://upload.wikimedia.org/wikipedia/commons/0/02/SVG_logo.svg" />
+      );
+      expect(prettyHtml(component.html())).toMatchSnapshot();
+    });
+
+    it('renders custom svg from relative url', () => {
+      const component = mount(<OuiIcon type="./assets/beaker.svg" />);
+      expect(prettyHtml(component.html())).toMatchSnapshot();
+    });
```

Comment:
nit - maybe it's better to test custom svg rendering with an asset that isn't used as a fallback.

Reply:
@joshuarrrr should I raise an issue so that we can revise this unit test or just leave it as it is?

Reply:
Sure, if you think it's a worthwhile change.